atc/db/pipeline.go

```diff
@@ -653,6 +654,40 @@ func (p *pipeline) Rename(name string) error {
 	return err
 }
 
+func (p *pipeline) Move(destinationTeamName string) error {
+	destinationTeamID, err := psql.Select("id").
+		From("teams").
+		Where(sq.Eq{
+			"name": destinationTeamName,
+		}).
+		RunWith(p.conn).Query()
```

Comment:
`Query()` can return no rows without throwing an error so maybe we should use `QueryRow()` instead and then use `Scan(&destinationTeamID)` to grab out the team id. We should always be expecting a team id to return from this query so we should be erroring if there is no rows being returned here.